modules/swagger-models/src/main/java/io/swagger/v3/oas/models/examples/Example.java

```diff
@@ -78,10 +80,11 @@ public Object getValue() {
 
     public void setValue(Object value) {
         this.value = value;
+        valueSetFlag = true;
```

Comment:
see how this is done in [Schema](https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-models/src/main/java/io/swagger/v3/oas/models/media/Schema.java#L690-L700) to correctly handle all scenarios